on-call/justfile

```
@@ -0,0 +1,22 @@
+set fallback
+set shell := ["bash", "-euxc", "-o", "pipefail"]
+
+# Connect to the postgres DB. Need to export PROFILE_PROD=SessionManagerForDeveloper-... 
+connect-pg:
+  #!/usr/bin/env bash
+  session-manager-plugin --version
+  if [ $? != "0" ]; then
+    echo "ERROR: SESSION MANAGER PLUGIN NOT INSTALLED "
+    exit 1;
+  fi
+  just --justfile ../b2b_personalization/justfile connect-pg "" prod
```

would be nice if it also works for dev and test. maybe we can just make the environment variable configurable

---

Good point!

---

```suggestion
  PROFILE_PROD=SessionManagerForDeveloper-987436120296 just --justfile ../b2b_personalization/justfile connect-pg "" prod
```
then we can actually simplify a bit with the profile